src/helperFunctions/hash.py

```diff
@@ -32,7 +32,7 @@ def get_tlsh_comparison(first, second):
     return tlsh.diff(first, second)
 
 
-def normalize_lief_items(functions):
+def normalize_lief_items(functions) -> list[str]:
```

Comment:
Couldn't we remove this function completely?
Why is this in `helperFunctions.hash` anyways?

Reply:
removed (or rather inlined)